infrastructure/tests/test_data_import.py

```diff
@@ -124,3 +126,63 @@ def test_year_error(self):
         self.assertEquals(AnnualSpendFile.objects.all().count(), 1)
         spend_file = AnnualSpendFile.objects.first()
         self.assertEquals(spend_file.status, AnnualSpendFile.ERROR)
+
+
+    def test_upload_project(self):
+        """Scope of Test: With no existing projects run an upload and check that the correct fields are populated"""
+        self.assertEquals(BudgetPhase.objects.all().count(), 5)
+
+        with open('infrastructure/tests/test_files/test.xlsx', 'rb', ) as f:
+            utils.load_excel("", financial_year="2019/2020", file_contents=f.read())
+
+        project = Project.objects.first()
+        self.assertEquals(project.function, "Administrative and Corporate Support")
+        self.assertEquals(project.project_description, "P-CNIN FURN & OFF EQUIP")
+        self.assertEquals(project.project_number, "PC002003005_00002")
+        self.assertEquals(project.project_type, "New")
+        self.assertEquals(project.mtsf_service_outcome, "An efficient, effective and development-oriented public service")
+        self.assertEquals(project.iudf, "Growth")
+        self.assertEquals(project.own_strategic_objectives, "OWN MUNICIPAL STRATEGIC OBJECTIVE")
+        self.assertEquals(project.asset_class, "Furniture and Office Equipment")
+        self.assertEquals(project.asset_subclass, "")
+        self.assertEquals(project.ward_location, "Administrative or Head Office")
+        self.assertEquals(project.longitude, 0.0)
+        self.assertEquals(project.latitude, 0.0)
+
+        self.assertEquals(Expenditure.objects.all().count(), 9)
+        expenditure = Expenditure.objects.first()
+        self.assertEquals(str(expenditure.project), " - P-CNIN FURN & OFF EQUIP")
+        self.assertEquals(str(expenditure.budget_phase), "Audited Outcome")
```

Comment:
```suggestion
        self.assertEquals(expenditure.budget_phase.name, "Audited Outcome")
```
would be more appropriate. See comment above.

Reply:
As above no longer using str